be/src/util/thrift_rpc_helper.cpp

```diff
@@ -68,11 +68,7 @@ Status ThriftRpcHelper::rpc_impl(const std::function<void(ClientConnection<Front
         callback(client);
         return Status::OK();
     } catch (apache::thrift::protocol::TProtocolException& e) {
```

Comment:
`TProtocolException` catch is not necessary

Reply:
fixed